ci-operator/config/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main__jaeger-product-hypershift-4.13-lp-rosa-hypershift.yaml

```
@@ -13,7 +13,7 @@ releases:
     candidate:
       product: ocp
       stream: nightly
-      version: "4.13"
+      version: "4.14"
```

```suggestion
      version: "4.13"
```

This job is meant to test rosa-hypershift which is specific to 4.13 for now, this change on line 17 isn't needed.